pagerduty/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM komand/python-pypy3-plugin:2
+FROM komand/python-3-37-slim-plugin:3
```

Comment:
```suggestion
FROM rapid7/insightconnect-python-3-38-slim-plugin:4
```

Reply:
Bump to runtime v4 - reach out to @ncramer-r7 on updating the rest of the plugin to accommodate this. Getting this on v4 will allow us to convert this to a cloud plugin :)